rgw/v2/tests/s3_swift/configs/test_Mbuckets_with_Nobjects_failed.yaml

```diff
@@ -0,0 +1,21 @@
+# upload type: non multipart
+# script: test_Mbuckets_with_Nobjects.py
```

Comment:
@TejasC88 , could you please add the description of read-only site

Reply:
Done